server/routes/temporaryabsences/temporaryAbsencesController.test.ts

```diff
@@ -58,6 +58,9 @@ afterEach(() => {
 
 describe('GET /prisoners-returning', () => {
   it('should render /prisoners-returning page with correct title when supportingMultitransactionsEnabled is true', () => {
+    config.supportingMultitransactionsEnabled = true
+    app = appWithAllRoutes({ services: { temporaryAbsencesService }, roles: [Role.PRISON_RECEPTION] })
```

Comment:
This test passed locally without line 61-61 but `config.supportingMultitransactionsEnabled = true` and app needed to make this test pass on circle